qabelbox/src/main/java/de/qabel/desktop/repository/sqlite/hydrator/SimpleContactHydrator.kt

```diff
@@ -0,0 +1,45 @@
+package de.qabel.desktop.repository.sqlite.hydrator
+
+import de.qabel.core.config.Contact
+import de.qabel.core.crypto.QblECPublicKey
+import de.qabel.core.drop.DropURL
+import de.qabel.desktop.repository.EntityManager
+import de.qabel.desktop.repository.sqlite.schemas.ContactDB
+import org.spongycastle.util.encoders.Hex
+import java.sql.ResultSet
+import java.util.*
+
+class SimpleContactHydrator(private val entityManager: EntityManager) : AbstractHydrator<Contact>() {
+
+    override fun hydrateOne(resultSet: ResultSet?): Contact? {
+        var column = 1
+        val id = resultSet!!.getInt(column++)
```

Comment:
No !! please. Either you don't accept null values officially in the signature or you handle null values.

A simple way to return null if your argument is null:
`resultSet ?: return null`
as the first line in the method.